vertx-jooq-shared-reactive/src/main/java/io/github/jklingsporn/vertx/jooq/shared/reactive/JsonAccessor.java

```diff
@@ -13,16 +12,17 @@
 
     private JsonAccessor(){}
 
-    public static JsonObject getJsonObject(Row row, String field){
-        return (JsonObject) getJson(row,field).value();
-    }
-
-    public static JsonArray getJsonArray(Row row, String field){
-        return (JsonArray) getJson(row,field).value();
-    }
-
-    public static Json getJson(Row row, String field) {
-        return row.getJson(field) == null ? JsonImpl.NULL : row.getJson(field);
-    }
+//    public static JsonObject getJsonObject(Row row, String field){
```

Comment:
ok so this class.. the generic Row interface does't have a getJson method. The postgres implementation does, but not entirely sure how to go about retaining this feature